src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -322,7 +322,7 @@ public void execute(Boolean succeeded)
       deps.add(Dependency.cranPackage("R6", "2.0"));
       deps.add(Dependency.cranPackage("sourcetools", "0.1.5"));
       deps.add(Dependency.cranPackage("htmltools", "0.3.5"));
-      deps.add(Dependency.cranPackage("shiny", "0.13", true));
+      deps.add(Dependency.cranPackage("shiny", "1.0"));
```

Comment:
Why did we formerly require installing shiny as a source package? (don't actually recall, just curious why it's no longer necessary!)

Reply:
I believe that, at the time, binary packages for v0.13 of Shiny were not available on CRAN, so we tried to force installation from sources.